pkg/sql/logictest/testdata/logic_test/materialized_view

```diff
@@ -29,6 +29,15 @@ SELECT * FROM v
 3 4
 5 6
 
+let $orig_crdb_timestamp
+SELECT max(crdb_internal_mvcc_timestamp) FROM v
+
+# All the rows have the same crdb_internal_mvcc_timestamp.
+query I
+SELECT count(*) FROM v WHERE crdb_internal_mvcc_timestamp != $orig_crdb_timestamp
```

Comment:
The fact that all rows have the same timestamp prior is just an accident of batching, and not something I think we want to assume. I'm not sure we want this assertion.

Reply:
ah good to know. i'll just take it out; i don't see a need to check anything here then